recipes-extended/netconfig/files/localalias.network

```diff
@@ -0,0 +1,9 @@
+[Match]
+Name=lo
+
+[Network]
+
+[Address]
+Label=lo:1
+Address=10.0.0.7/24
+[Route]
```

Comment:
Missing the EOL at EOF.

Reply:
Commit message is updated.
EOL at EOF is added.